ebssurrogate/scripts/surrogate-bootstrap.sh

```diff
@@ -211,6 +211,9 @@ function update_systemd_services {
 
 	# Disable postgresql service during first boot.
 	rm -f /mnt/etc/systemd/system/multi-user.target.wants/postgresql.service
+
+	# Disable fail2ban service during first boot.
+	rm -f /mnt/etc/systemd/system/multi-user.target.wants/fail2ban.service
```

Comment:
systemd disable in `setup-fail2ban`?

Reply:
hmm is there a reason to handle even e.g. postgres here instead of where we're setting it up initially?

Reply:
ok, disabled it via ansible task. Ansible task for postgres shows `enabled: yes`